units/commit_queue.go

```diff
@@ -296,26 +295,6 @@ func (j *commitQueueJob) TryUnstick(ctx context.Context, cq *commitqueue.CommitQ
 	}
 
 	// patch is done
-	if !utility.IsZeroTime(patchDoc.FinishTime) {
```

Comment:
Why is this removed? Is it impossible for `patchDoc.FinishTime` to be zero?

Reply:
I'm not sure how this got removed.. possibly residual local changes from a different ticket or a bad merge. Re-added

Reply:
I see what happened. When theres a redundant return the linter underlines it in my IntelliJ IDE and offers a suggestion to remove it automatically for you. Sometimes if you click "yes" the return statement and the above logic block gets removed along with it. I must have just clicked for auto-removal and not noticed this logic went away with it. Moral of the story do not use that feature in IntelliJ and just delete the statement with your keyboard haha.